packages/components/src/components/desktop-wrapper/desktop-wrapper.tsx

```diff
@@ -1,6 +1,7 @@
+import { PropsWithChildren } from 'react';
 import { isDesktop } from '@deriv/shared';
 
-const Desktop = ({ children }) => {
+const Desktop = ({ children }: PropsWithChildren<never>) => {
```

Comment:
@hamid-deriv why did you use `never` ?

Reply:
@iman-fs `PropsWithChildren` is a generic type that requires additional types to be passed. I pass never here because there is just a `children` prop here and doesn't accept any other props.

Reply:
type TDesktopProps = { children: React.ReactNode };

Let's stick to the type that we used for children in the whole project.
React.ReactNode is the recommended type for children prop.

Reply:
@hamid-deriv, wouldn't it be better to type the `children` prop `React.ReactElement` | `React.ReactElement[]` Closing this, because it's been merged to the parent branch: https://redmine.deriv.cloud/issues/77140#Jim/_Task_-_components_and_shared_ts_migration_FE_Core_Sprint_05/09/2022_(05_Sep_-_26_Sep) of `never`?

Reply:
@hamid-deriv Actually I tested it in a sample code.the problem is where you use it: 

example 1: 
```
          <Test>
              test
            </Test>
            or
            <Test>
              <div>test</div>
            </Test>
        //returns error: This JSX tag's 'children' prop expects type 'never' which requires multiple children, but only a single child was provided.
```

Reply:
Changed :+1: